dotnet/README.md

```diff
@@ -121,31 +121,29 @@ more features, such as connectors to services and Skills to perform specific act
 Unless you need to optimize which packages to include in your app, you will usually
 start by installing this meta-package first:
 
-* **Microsoft.SemanticKernel**
+* **[Microsoft.SemanticKernel](https://www.nuget.org/packages/Microsoft.SemanticKernel/#readme-body-tab)**
```

Comment:
Going to remove the "#readme-body-tab", as it scrolls to the bottom of the page and past all the version info, install links, etc.

```suggestion
* **[Microsoft.SemanticKernel](https://www.nuget.org/packages/Microsoft.SemanticKernel/)**
```